python/generate_asciipb.py

```diff
@@ -0,0 +1,49 @@
+from google.protobuf.text_format import PrintMessage
+from math import sqrt
+import sys
+sys.path.insert(1, 'proto/py')
```

Comment:
this is very bad practice in python.
you should rather add the following empty` __init__.py` files:
`python/__init__.py`
`proto/__init__.py`
`proto/py/__init__.py`

Then to call this script, from the root folder run
```
python3 -m python.generate_asciipb
```

Reply:
It doesn't work. Even after prefixing the pb2 imported files with ```proto.py.```, the following error is raised: ```No module named 'distribution_pb2'```

Reply:
I just tried on my system, it works if I import them as
```
from proto.py import mixing_prior_pb2
from proto.py import hierarchy_prior_pb2
```

Reply:
Still doesn't work :neutral_face: I've pushed the files, please try it yourself

Reply:
Have you ran 2to3 and added the __init__.py files? Please add to git also those

Reply:
The ```__init.py``` are there. 2to3 was also ran when generating the protos. Should I re-run the script after adding these fake submodules?

Reply:
Here are the exact commands I ran on my laptop from the bayesmix root folder
```
rm -rf proto/py/*
rm -rf proto/cpp/*
git pull upstream priors
./bash/compile_protos.sh
python3 -m python.generate_asciipb
```
and it works

Reply:
I literally copy-pasted them and I get the same error as before...

Reply:
Please send me an email with the output of all the commands

also include the output of 'which python3' 'python3 --version' and 'tree -L 2'

Reply:
What. When I re-run ```./bash/compile_protos.sh``` separately for a second time, it worked.

Oh well.